src/fsevent.rs

```diff
@@ -208,8 +208,8 @@ impl Watcher for FsEventWatcher {
       fsevent = FsEventWatcher {
         paths: cf::CFArrayCreateMutable(cf::kCFAllocatorDefault, 0, &cf::kCFTypeArrayCallBacks),
         since_when: fs::kFSEventStreamEventIdSinceNow,
-        latency: 0.1,
-        flags: fs::kFSEventStreamCreateFlagFileEvents,
+        latency: 0.0,
+        flags: fs::kFSEventStreamCreateFlagFileEvents | fs::kFSEventStreamCreateFlagNoDefer,
```

Comment:
Defer will wait several event to aggregate them and deliver to the callback. This can mess tests (at least) and this seems to be a reasonable default, until somebody tells us this messes things. Same goes with `latency`.